src/klein/_form.py

```diff
@@ -453,7 +452,8 @@ def defaultValidationFailureHandler(
 _requirerFunctionWithForm = Any
 _routeCallable = Any
 _routeDecorator = Callable[
-    [_routeCallable, DefaultNamedArg(Any, "__session__")], _routeCallable
+    [_routeCallable, DefaultNamedArg(Any, "__session__")],  # noqa: F821
+    _routeCallable,
```

Comment:
These type aliases should go away.  See https://github.com/twisted/klein/issues/414